Utilities/Thread.cpp

```diff
@@ -1865,9 +1865,8 @@ void thread_base::detach()
 
 void thread_base::notify()
 {
-	if (!(m_signal & 1))
```

Comment:
Not a bug, not a fix.

Reply:
if flag turned on between the if and the OR on other thread, it didnt detect it.

Reply:
Not a bug, not a fix. Don't make me repeat myself, or I'll just close the PR.